### ISSUE TITLE
Apparently Oracle doesn't understand retirement.

### ISSUE THREAD

**Issue by djTheRed** [mguessan/davmail#334]
DavMail requires the Java SDK. Currently, Oracle is not letting me install the SDK because I don't have a work email address. I'm meeting with them later today but not holding out much hope based on the info I've received from them so far. I am currently working on setting up my third server and need an email/calendaring, etc. solution for my family. 

Can you all rethink the way you are treating retirees?

I'd like to try DavMail but I'm close to giving up. That is a shame as my grandkids will remember this far into the future.

Cheers,

dj

**Comment by esabol** [comment]
Hi, @djTheRed. You don't need to use Oracle. OpenJDK is an open source version. https://openjdk.org/install/

**Comment by djTheRed** [comment]
Hi Ed and  mguessan,

I apologize for bothering  you both. At this point I probably need pyschological help as much as Davmail/computer help.

I believe I have installed  a version of the JDK that refers to itself as "Zulu", some  flavor of "21".  I've used Linux/Unbuntu/Soloaris, etc. as little  as possible in my life, it just appears to be a very tall mountain and having been a Windows user since v3.0, I still find Windows challenging. There is no Zulu program or shortcut  but there is a Zulu folder full of Linux sounding items. Actually, I believe during this adventure, I installed it twice,. There is a Zulu app listed under installed Apps in my Windows Server 2019  "Apps" section.

Davmail does not seem impressed and still claims I do not have an "suitable"  version of the JDK on my system. I did get a little further during install of Davmail and he put a listener in my tray for a while but it appears he is no longer there. Davmail is listed as an installed app in Windows/settings. I attempted to run Davmail from the command line and that was rejected as an unkown command/program.

I found the Davmail install instructions a little confusing and I'm no longer certain what all I did. I do recall at one I was installing some version of 6.blah.blah and at another point it was 1.blah.blah. Then I was a little confused installing the "no-install" version.  There were a lot of configuration questions at the end of the whole process that I didn't have answers for.

I don't consider myself completely inept. Once upon a time I installed and administered a cc:Mail post office back when email was an infant. I do realize things have become quite more complex since then.

My goal/expectations: I'm putting together a small family network so I can teach my grandkids about it. I currently have Win 2012r12 and Win2019 servers up and running although I'm still configuring them. A Ubuntu server is in the works but trying to get further along with the Windows before trying that. I bought two Domains and have done nothing with them yet. I need to configure static IPs before I can complete setting up my Domain(s). That apparently will require discussion with Comcast/XFinity that I'm not looking forward to.

Bottom line: I'm looking for a simple email server(free) for my family post office/calendaring/yada//yada.  Am I just in the wrong place for that? I have found this whole process much more daunting/frustrating than anticipated. No one writes straight forward installation instructions anymore.  And no offense to Davmail but there were what looks like 2 or 3 different starting points and configuration appears pretty complex.

I apologize that this email is so long. No one to vent to around here.

Do you think I should continue with trying Davmail  or is it time to move on?

Thanks,

dj



________________________________

**Comment by esabol** [comment]
@djTheRed asked:
> Bottom line: I'm looking for a simple email server(free) for my family post office/calendaring/yada//yada.  Am I just in the wrong place for that?

It sounds like it. DavMail is a gateway server for making Microsoft Exchange servers accessible to open standards like POP, IMAP, and SMTP. So unless you already have a Microsoft Exchange server that you're trying to access, then DavMail won't help you.

**Comment by esabol** [comment]
This issue can be closed, imho.

**Comment by mguessan** [comment]
Closing issue
